common/api-review/auth.api.md

```diff
@@ -767,6 +767,7 @@ export interface TotpMultiFactorInfo extends MultiFactorInfo {
 export class TotpSecret {
     readonly codeIntervalSeconds: number;
     readonly codeLength: number;
+    readonly enrollmentCompletionDeadline: string;
```

Comment:
Don't understand why it was changed to this instead of finalizeEnrollmentBy

Reply:
This is still pending API review, but IMO enrollmentCompletionDeadline sounds slightly clearer.